js/web/docs/webnn-operators.md

```diff
@@ -0,0 +1,96 @@
+## Operators Support Table
+
+The following table shows ONNX
+operators and the supported opset domain/versions in **WebNN EP** by ONNX Runtime Web. For example,
+`7-12, 13+` means ONNX Runtime Web currently supports opset version 7 to 12, 13 and above.
+
+(**Note**: ONNX Runtime only *guarantees* support for models stamped with opset version 7 or above for opset domain 'ai.onnx'.)
+
+[WebNN API](https://webmachinelearning.github.io/webnn) provides two device types `cpu` and `gpu` to leverage different on-device accelerators. WebNN API implementation in Chromium uses XNNPack backend for `cpu` device type and DirectML backend for `gpu` device type. The op support status behind these two backends is inconsistent since XNNPack backend currently supports limited op scope.
+
+
+| Operator | Opset | WebNN API | WebNN CPU | WebNN GPU | Comments |
+|:------:|:------:|:------:|:-:|:-:|:------|
+| Abs | ai.onnx(7-12, 13+) | abs | ✓ | ✓ | |
+| Add | ai.onnx(7-12, 13, 14+) | add | ✓ | ✓ | |
+| ArgMax | ai.onnx(7-10, 11, 12, 13+) | argMax | ✗ | ✓ | |
+| ArgMin | ai.onnx(7-10, 11, 12, 13+) | argMin | ✗ | ✓ | |
+| AveragePool | ai.onnx(7-9, 10, 11, 12-18, 19+) | averagePool2d | ✓ | ✓ | Only supports 4-D input, 2-D 'kernel_shape', 'count_include_pad' value is 0 |
```

Comment:
Should we document WebNN CPU only supports limited nhwc?

Reply:
Do you mean input layout? Don't worry, CPU backend in WebNN EP uses preferred layout nhwc, ORT will take care of this.